Library/Formula/cycript.rb

```diff
@@ -0,0 +1,91 @@
+class Cycript < Formula
+  desc "Modify apps using a hybrid of Objective-C++ and JavaScript syntax."
+  homepage "http://www.cycript.org"
+  url "git://git.saurik.com/cycript.git",
+    :tag => "v0.9.592",
+    :revision => "97bec96b43b66ab78af95d2b6c6c24f0d0a8006a"
+
+  head "git://git.saurik.com/cycript.git"
+
+  depends_on "bison" => :build
+  depends_on "android-sdk" => :build
+  depends_on "readline"
+
+  patch :DATA
+
+  def install
+    ENV.append "CPPFLAGS", "-Wno-inconsistent-missing-override"
```

Comment:
Clang seems to have added some kind of new warning. I guess I should really disable -Wall if you are not in maintainer mode, as otherwise every time someone uses a compiler different than one of the ones I have directly tested with, the build is subject to randomly fail. (I am used to just telling people what compilers I am compatible with, but that is no longer a viable situation for Cycript, and is no longer necessary since I moved all the insane parts out to a different in-coupled project.) I'll fix that at some point.